### PR TITLE
new package zsh-completions

### PR DESCRIPTION
**Pull request by Freed-Wu**

### CODE REVIEW

**Comment by Grimler91** [comment]
Have you checked so that none of the existing zsh completions conflict with this one? 

Please add `TERMUX_PKG_RECOMMENDS=zsh-completions` to zsh also

**Comment by Freed-Wu** [comment]
Do you mean change `TERMUX_PKG_DEPENDS=zsh` to `TERMUX_PKG_RECOMMENDS=zsh`?
However, bash-completion uses `TERMUX_PKG_DEPENDS=bash`. Is there any difference?

**Comment by Freed-Wu** [comment]
Oh, I get, you means change zsh.

**Comment by Grimler91** [comment]
> Oh, I get, you means change zsh.

Yes, since zsh-completions enhance zsh it would be nice to have it recommend or suggest zsh-completions

**Comment by Freed-Wu** [comment]
zsh-completions will not conflict with the existed zsh completions because they
don't accept these completions existed in other programs.

File: packages/zsh-completions/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/zsh-users/zsh-completions
+TERMUX_PKG_DESCRIPTION="Additional completion definitions for Zsh"
+TERMUX_PKG_LICENSE=BSD
```
Comment (xtkoba) [inline packages/zsh-completions/build.sh:3]: The license is Zsh license, not `BSD`. Specify `custom` for `TERMUX_PKG_LICENSE`.

**Review by xtkoba — CHANGES_REQUESTED** [review]
`TERMUX_PKG_RECOMMENDS` affects package metadata. Please bump `TERMUX_PKG_REVISION` (for `bash` and `zsh`).

**Comment by Freed-Wu** [comment]
OK.

File: packages/bash/build.sh [hunk 2]
```diff
@@ -5,10 +5,11 @@ TERMUX_PKG_MAINTAINER="@termux"
 _MAIN_VERSION=5.1
 _PATCH_VERSION=16
 TERMUX_PKG_VERSION=${_MAIN_VERSION}.${_PATCH_VERSION}
+TERMUX_PKG_REVERSION=2
```
Comment (xtkoba) [inline packages/bash/build.sh:8]: Should be `TERMUX_PKG_REVISION=1`

File: packages/zsh/build.sh [hunk 3]
```diff
@@ -4,11 +4,13 @@ TERMUX_PKG_LICENSE="custom"
 TERMUX_PKG_LICENSE_FILE="LICENCE"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=5.8.1
+TERMUX_PKG_REVERSION=2
```
Comment (xtkoba) [inline packages/zsh/build.sh:7]: Should be `TERMUX_PKG_REVISION=1`

**Review by xtkoba — CHANGES_REQUESTED** [review]

File: packages/zsh-completions/build.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/zsh-users/zsh-completions
+TERMUX_PKG_DESCRIPTION="Additional completion definitions for Zsh"
+TERMUX_PKG_LICENSE=custom
```
Comment (xtkoba) [inline packages/zsh-completions/build.sh:3]: Needs `TERMUX_PKG_LICENSE_FILE`:

```bash
TERMUX_PKG_LICENSE_FILE="LICENSE"
```

**Review by xtkoba — CHANGES_REQUESTED** [review]

**Comment by Freed-Wu** [comment]
OK.

**Review by xtkoba — CHANGES_REQUESTED** [review]
Not `TERMUX_PKG_REVERSION` but `TERMUX_PKG_REVISION`.

**Comment by Freed-Wu** [comment]
ok.

**Review by xtkoba — APPROVED** [review]

**Pull request merged**